pkg/storegateway/stats.go

```diff
@@ -70,12 +67,13 @@ type queryStats struct {
 	streamingSeriesWaitBatchLoadedDuration time.Duration
 
 	// The Series() request timing breakdown.
-	streamingSeriesExpandPostingsDuration       time.Duration
-	streamingSeriesFetchSeriesAndChunksDuration time.Duration
-	streamingSeriesEncodeResponseDuration       time.Duration
-	streamingSeriesSendResponseDuration         time.Duration
-	streamingSeriesOtherDuration                time.Duration
-	streamingSeriesIndexHeaderLoadDuration      time.Duration
+	streamingSeriesExpandPostingsDuration  time.Duration
+	streamingSeriesEncodeResponseDuration  time.Duration
+	streamingSeriesSendResponseDuration    time.Duration
+	streamingSeriesIndexHeaderLoadDuration time.Duration
+
+	// streamingSeriesAmbientTime is the total WAL time spent serving the request. It includes all other durations.
```

Comment:
Is this supposed to be `wall`, not `WAL`?
```suggestion
	// streamingSeriesAmbientTime is the total wall clock time spent serving the request. It includes all other durations.
```